### DISCUSSION TITLE
Requests & Questions

### DISCUSSION THREAD

**Discussion by zstenger93** [zstenger93/zUI#1] [Q&A] [answered]
Here you can ask for your favorite missing settings

**Comment by radhaz** [comment]
[REQUEST] A profile option so I can more easily use this across all my toons.

**Reply by zstenger93** [reply]
With the profile option you mean to have the option just instantly apply the same settings to other characters by selecting the desired profile of another character?

**Reply by radhaz** [reply]
Yes, I play a lot of different characters and endeavor to set each characters UI up identically so having a profile system where I could either make a "global" default option or just profiles that I could save would be useful especially as more features get added to the addon.

**Reply by zstenger93** [reply]
Cool, just asked to make sure. I throw it on the to-do list. Someone also asked for different settings for different specs on the character.
These are good improvements which will be done, with time.
The focus is now on finding this annoying event error and a few smaller issues then I can move on to stuff like these.
Thanks for the donation btw, I'm glad you like it that much! :)

**Reply by zstenger93** [reply]
Added also this one, tho took quite a while. Now you should be able to apply between any other character's profile to the current one. Let me know how is it, if all is good just mark this as an answer too.

**Comment by radhaz** [comment]
[REQUEST] An "option" in the config to show keybindings for unassigned hotbar keys. Like have the text for the keybindings there and probably padded/stroked with a darker color to allow for easier assigning of spells to hotbars and then the user can turn off the option once everythings set for them?

**Reply by zstenger93** [reply]
With this one, I'm slightly confused what is the idea.
Do you mean showing in a panel the available keybindings you could set? Or if a spell bar slot has no binding, showing a list of keybindings you could set there?
Or am I getting it completely wrong

**Reply by radhaz** [reply]
I'll try to explain better. So I made a new toon and as such I dragged the spells to the hotbar, I want certain spells on qwa and others on rtg, the problem arises that if I have no spells in the slot on the hotbar I can't see which button on the hotbar is qwa/rtg etc.  The feature I'm asking for is an option to toggle on all the set keybinds on every button on the hotbar an alternative that would probably be superior but more complex would be whenever a spell is dragged off the spell book then the keybinds automatically showed on the buttons.

**Reply by zstenger93** [reply]
Ahh gotcha, so just showing the hotkeys set on the slots, whenever you drag something like spells from the spellbook to the action bar. Should be fairly easy, since I do show borders when you drag something, probably can add it there. Will take a look at it.

**Reply by zstenger93** [reply, marked as answer]
Added, the new version is under review on curseForge. (on weekends they sometimes slow with the process)
Let me know if it works well for you. (in case you drag from the spellbook it has a bit of delay for some reason atm)
If all is good, please mark this as answer
